site.config.js

```diff
@@ -49,11 +49,7 @@ export default {
 
   // map of notion page IDs to URL paths (optional)
   // any pages defined here will override their default URL paths
-  // example:
-  //
-  // pageUrlOverrides: {
-  //   '/foo': '067dd719a912471ea9a3ac10710e7fdf',
-  //   '/bar': '0be6efce9daf42688f65c76b89f8eb27'
-  // }
-  pageUrlOverrides: null
+  pageUrlOverrides: {
+    '/devpool': 'fd3d3861654f4b7b9c2702ff7842ade2', // https://dao.ubq.fi/ubiquityos-for-daos-fd3d3861654f4b7b9c2702ff7842ade2
```

Comment:
```suggestion
    '/devpool': 'fd3d3861654f4b7b9c2702ff7842ade2', // https://dao.ubq.fi/ubiquityos-for-daos
```

I think it should be redirected to the one without the hash at the end can you redo?

Reply:
It should be fine, preview redirects to a page without any hashes https://dao-ubq-fi-git-fork-rndquu-feat-d-a0db76-eths-projects-4301eec5.vercel.app/devpool